package.json

```diff
@@ -16,12 +16,13 @@
     "jest": "^26.6.3",
     "webpack": "^5.23.0",
     "webpack-cli": "^4.5.0",
-    "webpack-dev-server": "^3.11.2"
+    "webpack-dev-server": "^3.11.2",
+    "webpack-merge": "^5.7.3"
   },
   "scripts": {
-    "start": "webpack serve --node-env development",
-    "build:dev": "webpack --node-env development",
-    "build:prod": "webpack --node-env production",
```

Comment:
hey @inductio my previous comment was not about deleting `webpack.config.js`, but to add `webpack.common.js` to it with common config for both parts.

What is required for AC is to have `--node-env` as you did previously and inside `webpack.config.js` do something like this:

```
const dev = require('./webpack.dev.config.js');
const prod = require('./webpack.prod.config.js');
const common = require('./webpack.common.config.js');

module.exports = process.env.NODE_ENV === 'development'
    ? merge(common, dev)
    : merge(common, prod)
```

Thus you benefit from `process.env.NODE_ENV`